app/validation/validator.py

```diff
@@ -29,7 +30,9 @@ def __init__(self, schema, validation_store, response_store):
             AbstractValidator.MANDATORY: _("This field is mandatory."),
             AbstractValidator.INVALID_DATE: _("This is not a valid date."),
             AbstractValidator.NEGATIVE_INTEGER: _("Negative values are not allowed."),
-            AbstractValidator.INTEGER_TOO_LARGE: _('This number is too big.')
+            AbstractValidator.INTEGER_TOO_LARGE: _('This number is too big.'),
+            AbstractValidator.INVALID_DATE_RANGE_DIFF: _("The 'to' date cannot be before the 'from' date."),
```

Comment:
Again should the words 'to' and 'from' be in these defaults?

Also should these error messages be present in the schema?